SearchEngineCS/Phase5/SearchLibrary/InvertedIndex.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SearchLibrary
+{
+    public class InvertedIndex
+    {
+        public Dictionary<string, HashSet<string>> Map { get; set; }
+
+        public InvertedIndex()
+        {
+            Map = new Dictionary<string, HashSet<string>>();
+        }
```

Comment:
- You don't need setter for `Map` property.
- You don't need this default constructor. use the following code instead:
```CSharp
 public Dictionary<string, HashSet<string>> Map { get; } = new Dictionary<string, HashSet<string>>();
```